docs/guides/run-primitives-rest-api.mdx

```diff
@@ -0,0 +1,489 @@
+             None # parameter values
+             ]],
+        "version": 2, # this defines the version of the Qiskit Runtime Primitive to use, c.f. https://docs.quantum.ibm.com/api/migration-guides/v2-primitives
+        "options": {
+            "dynamical_decoupling": { 
+                "enable": True,
+                "sequence_type": 'XpXm',
+                "extra_slack_distribution": 'middle',
+                "scheduling_method": 'alap',
+            },
+        },
+    }
+}
+
+response = requests.post(url, headers=headers, json=job_input)
+
+if response.status_code == 200:
+    job_id = response.json().get('id')
+    print("Job created:",response.text)
+else:
+    print(f"Error: {response.status_code}")
+```
+
+  </TabItem>
+
+  <TabItem value="Twirling" label="Twirling">
+
+  ```python
+import requests
+
+url = 'https://api.quantum-computing.ibm.com/runtime/jobs'
+headers = {
+    'Content-Type': 'application/json',
+    'x-access-token':auth_id,
+    'x-qx-client-application': 'qiskit-version-2/0.39.2/'+'your_application' # specifying the application you might be running from. For an actual Integration project, this option is invaluable to know where jobs are coming from. At this time the "qiskit-version-2/0.39.2/" string is a necessary prefix.
+    }
+job_input = {
+    'program_id': 'estimator',
+    "backend": backend, 
+    "hub": "hub1",
+    "group": "group1",
+    "project": "project1",
+    "start_session": "False", # set to False if you just need to run a single job.  
+    "params": {
+        "pubs": [ #primitive unified blocs (PUBs) containing one circuit each. c.f. https://docs.quantum.ibm.com/api/migration-guides/v2-primitives
+            [resulting_qasm, # QASM circuit
+             {"IIZII": 1, "XIZZZ": 2.3}, # Observable
+             None # parameter values
+             ]],
+        "version": 2, # this defines the version of the Qiskit Runtime Primitive to use, c.f. https://docs.quantum.ibm.com/api/migration-guides/v2-primitives
+        "options": {
+            "twirling": { 
+                "enable_gates": True, 
+                "enable_measure": True,
+                "num_randomizations": "auto",
+                "shots_per_randomization": "auto",
+                "strategy": "active-accum",
+                },
+        },
+    }
+}
+
+response = requests.post(url, headers=headers, json=job_input)
+
+if response.status_code == 200:
+    job_id = response.json().get('id')
+    print("Job created:",response.text)
+else:
+    print(f"Error: {response.status_code}")
+```
+
+
+  </TabItem>
+</Tabs>
+
+
+<span id="start-sampler"></span>
+## Sampler Primitive with REST API
+
+### 1. Initialize the account
+
+Because Qiskit Runtime `Sampler` is a managed service, you first need to initialize your account. You can then select the device you want to use to calculate the expectation value. You can then select the IBM Quantum Compute resource of your choice to run your calculations on. 
```

Comment:
Sampler doesn't calculate expectation values, right?...

Reply:
No, it doesn't. Thanks for catching that.